.gitignore

```diff
@@ -31,3 +31,5 @@ man/man1
 man/man5
 man/man8
 vendor/pkg/
+# generated by integration-cli-on-swarm
```

Comment:
Ah, sorry for confusing comment.
`docker-compose.yml` is generated by integration-cli-on-swarm, but `.gitignore` is still written by hand and I forgot to add line break.

I'll fix this (Actually no need to put yml there, I'll try to use /tmp/rand0m.yml instead)